package.json

```diff
@@ -13,6 +13,7 @@
   },
   "scripts": {
     "test": "grunt test",
+    "teststyle": "grunt teststyle",
     "generate-certificates": "node scripts/generate-certificates.js"
```

Comment:
this can be removed, unused since we switched to the plain HTTP server

Reply:
@siddharthvp Is this comment in reference to the line "generate-certificates"? So I should delete that line and also its associated file?

Reply:
Let the file remain for now, but yeah I think we can remove the npm script.